packages/bpk-component-graphic-promotion/src/BpkGraphicPromo-test.tsx

```diff
@@ -15,35 +15,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* @flow strict */
 
 import React from 'react';
 import { fireEvent, render } from '@testing-library/react';
+// @ts-expect-error Untyped import. Not added as a stub declaration so consumers still have the suppression in their node_modules when shipped untranspiled.
```

Comment:
Hi! I've read that above. Just to confirm, the stub declaration won't be shipped with the package, if the package is not transpiled?

Reply:
We can ship them, it's just if TypeScript pays attention to them.

If there is an `index.d.ts` per `package.json` then TypeScript will pay attention, or if a `types` field is added to each `package.json`, but not otherwise. We can make it work this way, but it is a much more hidden implementation, for both maintainers and consumers. The proposed solution (temporarily suppressing inline) is a bit more ugly as its so visible, but the fact it is visible makes it clearer what is going on, and has the benefit that as soon as real types are added the suppression will error to remind us to clean up - not risking us stubbing modules that actually do have types without realising.

Reply:
>If there is an index.d.ts per package.json then TypeScript will pay attention, or if a types field is added to each package.json, but not otherwise.

This won't exist anymore when we stop publishing individual packages as all the package files per component will be gone!